src/AndcultureCode.CSharp.Sitefinity.Conductors/Extensions/DynamicContentExtensions.cs

```diff
@@ -80,7 +77,12 @@ string providerName
             var sitefinityContentSubclass = property.PropertyType.IsSubclassOf(typeof(SitefinityContent));
             var contentSubclass = property.PropertyType.IsSubclassOf(typeof(Content));
 
-            if (!sitefinityContentSubclass && !contentSubclass)
+            // Enums must be adapted to a string value since Sitefinity stores them as a string
+            if (property.PropertyType.IsEnum)
+            {
+                dataItem.SetValue(property.Name, item.GetType().GetProperty(property.Name)?.GetValue(item)?.ToString());
+            }
+            else if (!sitefinityContentSubclass && !contentSubclass)
```

Comment:
The more of these `else if` statements we add the more I feel like we should just return within each `if` block. I feel like it's easier to parse that way.